config/prow/config.yaml

```diff
@@ -937,6 +937,8 @@ presets:
       value: capzci.azurecr.io
     - name: LOCAL_ONLY
       value: "false"
+    - name: USE_LOCAL_KIND_REGISTRY
```

Comment:
can you also add a comment to `LOCAL_ONLY` that it is there for backward compatibility reasons and should be removed once CAPZ v1.7.0+ is used in all jobs?